server/test/api/accountType.js

```diff
@@ -0,0 +1,89 @@
+/*global describe, it, beforeEach, process*/
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = chai.expect;
+chai.use(chaiHttp);
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+
+var url = 'https://localhost:8080';
+var user = { username : 'superuser', password : 'superuser', project: 1};
+
+describe('The /account_types API endpoint', function () {
+  var agent = chai.request.agent(helpers.baseUrl);
+  var newAccountType = {
+    id : 4,
+    type : 'test account type 1'
+  };
+
+  var DELETABLE_ACCOUNT_TYPE_ID = 3;
+  var FETCHABLE_ACCOUNT_TYPE_ID = 1;
+
+  // login before each request
+  beforeEach(helpers.login(agent));
+
+  it(' A GET /account_types returns a list of account type', function () {
+    return agent.get('/account_types')
+      .then(function (res) {
+        expect(res).to.have.status(200);
+        expect(res.body).to.not.be.empty;
+        expect(res.body).to.have.length(2);
+      })
+      .catch(helpers.handler);
+  });
+
+  it(' A GET /account_types/:id returns one account type', function () {
+    return agent.get('/account_types/'+ FETCHABLE_ACCOUNT_TYPE_ID)
+      .then(function (res) {
+        expect(res).to.have.status(200);
+        expect(res.body).to.not.be.empty;
+        expect(res.body.id).to.be.equal(FETCHABLE_ACCOUNT_TYPE_ID);
+      })
+     .catch(helpers.handler);
+  });
+
+  it('A POST /account_types will add an account_type', function () {
+    return agent.post('/account_types')
+      .send(newAccountType)
+      .then(function (res) {
+        expect(res).to.have.status(201);        
```

Comment:
You should immediately send a GET request verfiy that you can actually find this record in the database.  [See this for reference](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/test/api/users.js#L200).